docs/features/bonding-quick-start.md

```diff
@@ -0,0 +1,140 @@
+
+* Make the listener listen on every network interface (use the `INADDR_ANY` wildcard)
+* Make multiple listeners, each one bound to a different network interface
+
+In the second case, all you have to do is to configure multiple listener sockets. The one that gets the first member connection will report the group ID from the `srt_accept` call, all others will be handled in the background, regardless from which listener socket they were reported. For a blocking mode `srt_accept`, which cannot handle multiple listeners, there's a convenience function provided, `srt_accept_bond`. It can be provided with multiple sockets and it returns the first accepted socket that reports on any of the given listener sockets. In non-blocking mode you can still use it, it will simply find out which one is ready and report that one.
+
+**Use case.** Create multiple listeners bound to its own IP and port (NIC), but capable to accept connections within the same bonding group.
+
+Usage example:
+
+```c++
+vector<SRTSOCKET> listeners;
+
+for (int i = 0; i < 2; ++i)
+{
+    SRTSOCKET s = srt_create_socket();
+    const int gcon = 1;
+    srt_setsockflag(s, SRTO_GROUPCONNECT, &gcon, sizeof gcon);
+    srt_bind(s, (sockaddr*)&(sa[i]), sizeof (sa[i]));
+    srt_listen(s, 5);	listeners.push_back(s);
+}
+SRTSOCKET conngrp = srt_accept_bond(listeners.data(), listeners.size(), -1 /* msTimeOut */);
+```
+
+### 3.3. Getting Group Statistics and Status
+
+`srt_bistats(SRTSOCKET u, ...)` function can be used with a socket group ID as a first argument to get statistics for a group. Most values of the `SRT_TRACEBSTATS` will be filled with zeros except for the fields listed in [SRT Group Statistics Summary Table](../API/statistics.md#srt-group-statistics) below. Refer to the documentation of the [SRT API Functions](../API/API-functions.md#socket-group-management) for usage instructions.
+
+The number of socket members in a group can be determined by calling `srt_group_data()`. The code below retrieves the number of sockets in a group.
+
+```c++
+SRTSOCKET groupid = srt_create_group(SRT_GTYPE_BROADCAST);
+int group_size = 0;
+srt_group_data(groupid, NULL, &group_size);
+```
+
+To get information (status) on every member of the group at any time, you can use this function:
+
+```c++
+SRT_SOCKGROUPDATA output[group_size];
+srt_group_data(groupid, output, &group_size);
+```
+
+Now `output[i]` constains information on `i`-th socket. See [SRT_GROUP_DATA](../API/API-functions.md#srt_sockgroupdata) structure with [`SRT_MEMBER_STATUS`](../API/API-functions.md#srt_memberstatus) per socket.
+
+Note that broken member connections are automatically removed from the group and are no longer members. If you have your own array of endpoint definitions, you should keep it in your application and confront with the result reported by this function.
+
+## 4. FAQ
+
+TODO: To make proper formatting. Q = Question, A = Answer.
```

Comment:
@mbakholdina You could italicize the text for each question:

**Q.** *What is your name?*
**A.** My name is Steve.

Or use bulleted lists:

  - *What is your name?*
      - My name is Steve.